ksql-engine/src/test/java/io/confluent/ksql/EndToEndEngineTestUtil.java

```diff
@@ -315,14 +315,18 @@ public void configure(final Map<String, ?> properties, final boolean b) {
     private final String name;
     private final org.apache.avro.Schema schema;
     private final SerdeSupplier serdeSupplier;
+    private final int numPartitions;
```

Comment:
need to be able to control partition count from the json tests, otherwise some fail due to the stricter checking of partition counts in clients.